gnome-initial-setup/pages/password/gis-password-page.c

```diff
@@ -187,8 +186,7 @@ validate (GisPasswordPage *page)
   gtk_label_set_label (GTK_LABEL (priv->confirm_explanation), "");
   priv->valid_confirm = FALSE;
 
-  priv->valid_password = (strength_level > 1);
```

Comment:
(I merged this patch to avoid a lot of back-and-forth since it looks good, but it would have been simpler to just change this assignment to `TRUE` and keep the patch small.)